src/open_inwoner/accounts/models.py

```diff
@@ -324,31 +331,10 @@ class Meta:
                 & ~Q(login_type=LoginTypeChoices.eherkenning),
                 name="unique_email_when_not_digid_or_eherkenning",
             ),
-            # UniqueConstraint(
-            #     fields=["bsn"],
-            #     condition=Q(login_type=LoginTypeChoices.digid)
-            #     # not quite sure if we want this (bsn shouldn't be null AND blank)
-            #     & ~(Q(bsn="") | Q(bsn__isnull=True)),
-            #     name="unique_bsn_when_digid",
-            # ),
-            # UniqueConstraint(
-            #     fields=["rsin"],
-            #     condition=Q(login_type=LoginTypeChoices.eherkenning)
-            #     # not quite sure if we want this (rsin shouldn't be null AND blank)
-            #     & ~(Q(rsin="") | Q(rsin__isnull=True)),
-            #     name="unique_rsin_when_eherkenning",
-            # ),
-            # UniqueConstraint(
-            #     fields=["oidc_id"],
-            #     condition=Q(login_type=LoginTypeChoices.oidc),
-            #     name="unique_bsn_when_digid",
-            # ),
-            # CheckConstraint(
-            #     # maybe this is not correct?
-            #     check=(Q(bsn="") | Q(bsn__isnull=True))
-            #     | ~Q(login_type=LoginTypeChoices.digid),
-            #     name="check_digid_bsn_required_when_digid",
-            # ),
+            models.CheckConstraint(
+                check=~Q(phonenumber__exact="") | Q(phonenumber_alternative__exact=""),
```

Comment:
I was expecting:

```suggestion
                check=Q(phonenumber__exact="") & ~Q(phonenumber_alternative__exact=""),
```

That is: we should avoid a primary being unset _and_ a secondary being set.

Reply:
The check passes if the condition evaluates to `True` (or `Null`, which is not relevant though since the phone number fields are non-nullable). The condition here describes the allowed case: primary number is not empty or secondary is empty.

Reply:
Got it, confusion from my end, thanks.